packages/modules/cra-template-itwin-viewer/template.json

```diff
@@ -0,0 +1,55 @@
+{
+  "package": {
+    "dependencies": {
+      "@bentley/bentleyjs-core": "^2.6.4",
+      "@bentley/extension-client": "^2.6.4",
+      "@bentley/frontend-application-insights-client": "^2.6.4",
+      "@bentley/frontend-authorization-client": "^2.6.4",
+      "@bentley/geometry-core": "^2.6.4",
+      "@bentley/icons-generic": "^1.0.13",
+      "@bentley/imodelhub-client": "^2.6.4",
+      "@bentley/imodeljs-common": "^2.6.4",
+      "@bentley/imodeljs-frontend": "^2.6.4",
+      "@bentley/imodeljs-i18n": "^2.6.4",
+      "@bentley/imodeljs-markup": "^2.6.4",
+      "@bentley/imodeljs-quantity": "^2.6.4",
+      "@bentley/itwin-client": "^2.6.4",
+      "@bentley/itwin-viewer-react": "3.0.0",
+      "@bentley/orbitgt-core": "^2.6.4",
+      "@bentley/presentation-common": "^2.6.4",
+      "@bentley/presentation-frontend": "^2.6.4",
+      "@bentley/product-settings-client": "^2.6.4",
+      "@bentley/rbac-client": "^2.6.4",
+      "@bentley/react-scripts": "3.4.1",
+      "@bentley/telemetry-client": "^2.6.4",
+      "@bentley/ui-abstract": "^2.6.4",
+      "@bentley/ui-components": "^2.6.4",
+      "@bentley/ui-core": "^2.6.4",
+      "@bentley/ui-framework": "^2.6.4",
+      "@bentley/ui-ninezone": "^2.6.4",
+      "@bentley/webgl-compatibility": "^2.6.4",
+      "history": "^4.10.1",
+      "oidc-client": "^1.10.1",
+      "react": "16.13.1",
+      "react-dom": "16.13.1",
+      "@svgr/webpack": "4.3.3",
+      "@testing-library/jest-dom": "^4.2.4",
+      "@testing-library/react": "^10.4.4",
+      "@testing-library/user-event": "^7.1.2",
+      "@types/history": "^4.7.6",
+      "@types/jest": "^26.0.4",
+      "@types/node": "^14.0.19",
+      "@types/react": "^16.9.0",
+      "@types/react-dom": "^16.9.0",
+      "typescript": "^3.7.2"
+    },
+    "eslintConfig": {
+      "extends": ["react-app", "react-app/jest"]
+    },
+    "scripts": {
+      "start": "react-scripts --max_old_space_size=4096 start",
+      "build": "react-scripts --max_old_space_size=4096 build",
+      "test": "react-scripts --max_old_space_size=4096 test"
+    }
+  }
```

Comment:
Could we add a custom `browserlist` to the app?  We'd want it to match the minimum supported by iModel.js, which we currently have as,

```
"browserslist": [
    "electron 8.0.0",
    "last 4 chrome version",
    "last 4 firefox version",
    "last 4 safari version",
    "last 4 ios version",
    "last 4 ChromeAndroid version",
    "last 4 edge version",
    "not dead",
    "not <0.2%"
  ],
```

Reply:
we should be able to. I'll update

Reply:
added